package.json

```diff
@@ -31,11 +31,14 @@
     "react": "~16.9.0",
     "react-dom": "~16.9.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz",
+    "react-native-bottom-drawer": "^1.0.2",
     "react-native-geolocation-service": "^4.0.0",
     "react-native-google-maps-directions": "^2.1.1",
     "react-native-maps": "0.26.1",
     "react-native-screens": "2.0.0-alpha.12",
-    "react-native-web": "~0.11.7"
+    "react-native-vector-icons": "^6.6.0",
+    "react-native-web": "~0.11.7",
+    "rn-bottom-drawer": "^1.4.3"
```

Comment:
how come we have 2 bottom drawer libraries? Are all of these new libraries being used?

Reply:
yes, used by rn-bottom-drawer, that is the drawer i used

Reply:
yeah but we have 'rn-bottom-drawer' and 'react-native-bottom-drawer', are we using both of these